### ISSUE TITLE
Ladybug - Dynamo 2.1 vs 2.0

### ISSUE THREAD

**Issue by mengelmn** [ladybug-analysis-tools/ladybug-dynamo#39]
On different machines, I have a combination of Dynamo versions installed and active.  All Scripts and Packages behave normally with the exception of Ladybug.  

Ladybug operates correctly on the Machines with Dynamo Version 2.0.1 Installed
Ladybug returns an error message "Not Installed" on Machines running Dynamo 2.1

Primarily I see the error message on the "Sunpath" node.  It would appear it is not construction the path correctly to pull the python script.

Is there an update that would allow Ladybug to function on both 2.0 & 2.1?

**Comment by mostaphaRoudsari** [comment]
Hi @mengelmn, my best guess is that the dynamo team has changed the structure of the installation folder again! 😕 Can you share with me where the Ladybug package is installed when you use Dynamo 2.1?

**Comment by mengelmn** [comment]
Below is the path where it is installed on the machine with 2.1 The Only difference appears to be 2 vs 2.1  

C:\Users\__USER__\AppData\Roaming\Dynamo\Dynamo Core\2.1\packages

**Comment by mostaphaRoudsari** [comment]
Thanks! Are you not using it from inside Revit? It's interesting that it has been installed under Dynamo Core and not Dynamo Revit.

**Comment by mengelmn** [comment]
Correct, I'm currently using it via Dynamo Sandbox without Revit.  In the End, it will be applied to a Revit model, but design iterations have proved much faster when running separately from Revit

**Comment by mengelmn** [comment]
On a whim to get this functioning, I created a Dynamo 2.0 folder on the machine that does not work and placed the Ladybug Package in that location.  And it is now able to see the path and pull the .py files correctly.  This works for today, and also definitely reinforces its a pathing issue

**Comment by manum876** [comment]
I´m having the same error, but I'm running Dynamo Revit 2.0.
Says that it's not installed and that I should install it from package manager.

File locations are ok
